samples/batch/js/node/main.js

```diff
@@ -0,0 +1,236 @@
+const RECORDINGS_CONTAINER_URI = ""
+
+var callbackGetTrans = function(error, data, response) {
+  if (error) {
+    console.error(error);
+  } else {
+    console.log('Callback Get Transcription API called successfully. Returned data: ' + JSON.stringify(data));
+    completed=false ;
+    var tid = data.self.substring(data.self.lastIndexOf("/")+1);
+    console.log("Transcription Id " + tid);
+    status = _checkComplete(tid);
+
+    if (status == "Succeeded")
+    {
+        var opts = _getTranscriptionFiles();
+        apiInstance.getTranscriptionFiles(tid,opts,callbackGetTransFiles);
+    }
+    else if(data.status == "Failed")
+           console.log("Transcription failed: {transcription.properties.error.message}")
+  }
+};
+
+var callbackGetTransFiles = function(error, data, response) {
+  if (error) {
+    console.error(error);
+  } else {
+   console.log('callbackGetTransFiles API called successfully. Returned data: ' + JSON.stringify(data));
+   pag_files = data.values;
+   for (var i = 0; i < pag_files.length; i++)
+   {
+     var file_data = pag_files[i];
+     var url = file_data.links.contentUrl; 
+     if(file_data.kind !='Transcription')
+        continue;
+     else{
+        request(url, { json: true }, (err, res, body) => {
+            if (err) { 
+                return console.log(err); 
+            }
+            else {
+              console.log("\n\n\n Name of file : " + file_data.name);
+              console.log("\n Transcription File Content is : " + JSON.stringify(body));
+            }
+        });
+        
+     }
+   }
+  }
+};
+
+// uncomment to execute 
+// _transcribe();
+// _deleteAllTranscriptions();
+
+function _transcribe()
+{
+    
+    // Configure API key authorization: apiKeyHeader
+    var apiKeyHeader = defaultClient.authentications['apiKeyHeader'];
+    apiKeyHeader.apiKey = API_KEY;
+    // Configure API key authorization: apiKeyQuery
+    var apiKeyQuery = defaultClient.authentications['apiKeyQuery'];
+    apiKeyQuery.apiKey = API_KEY;
+
+    defaultClient.basePath = 'https:// centralindia.api.cognitive.microsoft.com/speechtotext/v3.0'.replace(/\/+$/, '');
+
+    // Uncomment this block to transcribe one file.
+    // var opts = _transcribeFromSingleBlob(RECORDINGS_BLOB_URI);
+
+    // Uncomment this block to transcribe all files from a container.
+    var opts = _transcribeFromContainer(RECORDINGS_CONTAINER_URI);
+    
+    apiInstance.createTranscription(opts, function(error, data, response) {
+        if (error) {
+          console.error(error);
+        } 
+        else {
+          var completed = false;
+          var tid = data.self.substring(data.self.lastIndexOf("/")+1);
+          console.log("Transcription ID created is : "+tid);
+          apiInstance.getTranscription(tid,callbackGetTrans);
+          
+        }
+      });
+    
+}// end of _transcribe()
+
+
+function _deleteAllTranscriptions(){
```

Comment:
This does not actually delete all transcriptions, because the GET transcriptions API returns transcription as pages (of 100 entities by default). Please mention this in a comment, and that it might be necessary to call this function multiple times; or implement the paged call.